docs/docs/examples/llm/maritalk.ipynb

```diff
@@ -60,11 +62,11 @@
     "from llama_index.core.llms import ChatMessage\n",
     "from llama_index.llms.maritalk import Maritalk\n",
     "\n",
+    "import asyncio\n",
+    "\n",
     "# To customize your API key, do this\n",
     "# otherwise it will lookup MARITALK_API_KEY from your env variable\n",
-    "# llm = Maritalk(api_key=\"<your_maritalk_api_key>\")\n",
-    "\n",
-    "llm = Maritalk()\n",
+    "llm = Maritalk(api_key=\"<your_maritalk_api_key>\")\n",
```

Comment:
Maybe pass the model name ("sabia-2-medium") here, so the user knows which model is being used?